src/sdk.ts

```diff
@@ -249,6 +250,45 @@ class RangeSDK implements IRangeSDK {
       return;
     }
 
+    // Filter out rate limited rules
+    const filteredRules: IRangeAlertRule[] = [];
+    for (const rule of rules) {
+      if (
+        this.rateLimitedRules.has(rule.id) &&
+        dayjs
+          .unix(this.rateLimitedRules.get(rule.id) as number)
+          .isAfter(dayjs())
```

Comment:
I believe you store the end timestamp (when the rule gets out of quarantine, right?) -> Then it should be .isBefore ...

Reply:
I default to pushing to filtered rules, so if this condition succeeds i just log for skipping the rule and `continue` out for this iteration.
Hence `isAfter` is used instead of `isBefore`